components/nifosg/matrixtransform.cpp

```diff
@@ -0,0 +1,20 @@
+#include "matrixtransform.hpp"
+
+namespace NifOsg
+{
+    MatrixTransform::MatrixTransform(int recordIndex, const Nif::Transformation &trafo)
```

Comment:
`trafo` is a weird abbreviation. If `transform` is too long, `trans` and `xform` are the conventional shortenings.

Reply:
NIF transformation is referred to as trafo in [many other instances within NIF code](https://github.com/OpenMW/openmw/search?q=trafo&unscoped_q=trafo), I'd rather not have them all corrected here.